app/controllers/api/v1/messages_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Api::V1::MessagesController < ApplicationController
+  def index
+    render json: Message.all
+  end
```

Comment:
- [ ] Kindly make the endpoint only return one random greeting message as per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/connect-frontend-frameworks/rails_react_webpack.md#:~:text=Create%20an%20API%20endpoint%20that%20selects%20a%20random%20greeting%20from%20your%20table).